klippa/src/glyf_loca.rs

```diff
@@ -0,0 +1,304 @@
+//! impl subset() for glyf and loca
+use crate::{estimate_subset_table_size, Plan, SubsetError, SubsetError::SubsetTableError};
+use write_fonts::{
+    read::{
+        tables::{
+            glyf::{
+                CompositeGlyph, CompositeGlyphFlags, Glyf,
+                Glyph::{self, Composite, Simple},
+                SimpleGlyph, SimpleGlyphFlags,
+            },
+            head::Head,
+            loca::Loca,
+        },
+        types::GlyphId,
+        FontRef, TableProvider, TopLevelTable,
+    },
+    FontBuilder,
+};
+
+pub fn subset_glyf_loca(
+    plan: &Plan,
+    font: &FontRef,
+    builder: &mut FontBuilder,
+) -> Result<(), SubsetError> {
+    let Ok(glyf) = font.glyf() else {
+        return Err(SubsetTableError(Glyf::TAG));
+    };
```

Comment:
not necessary but you could write this as, 
```rust
let glyf = font.glyf().ok_or(SubsetTableError(Glyf::TAG)?;
```

Reply:
thanks! This is very useful